src/Package/Impl/DataInspect/VisualGrid/VisualGrid.cs

```diff
@@ -65,36 +64,43 @@ internal sealed class VisualGrid : FrameworkElement {
             }
         }
 
-        public static readonly DependencyProperty FontSizeProperty =
-                TextElement.FontSizeProperty.AddOwner(
-                        typeof(VisualGrid));
+        public static readonly DependencyProperty FontSizeProperty = TextElement.FontSizeProperty.AddOwner(typeof(VisualGrid));
 
         [TypeConverter(typeof(FontSizeConverter))]
         public double FontSize {
             get { return (double)GetValue(FontSizeProperty); }
-            set { SetValue(FontSizeProperty, value); }
+            set {
+                _typeFace = null;
```

Comment:
DependencyProperty can be changed by WPF using only its metadata without direct access to the property. `DependencyProperty.AddOwner` has overload that allows to specify `FrameworkPropertyMetadata` instance and add custom `PropertyChangedCallback`